ydb/core/tx/schemeshard/schemeshard__operation_create_set_constraint.cpp

```diff
@@ -0,0 +1,19 @@
+#include "schemeshard__operation_common.h"
+#include "schemeshard__operation_part.h"
+#include "schemeshard_utils.h"  // for TransactionTemplate
+
+#include <ydb/core/protos/flat_scheme_op.pb.h>
+#include <ydb/core/protos/flat_tx_scheme.pb.h>
+#include <ydb/core/ydb_convert/table_description.h>
+
+namespace NKikimr {
+namespace NSchemeShard {
+    TVector<ISubOperation::TPtr> CreateSetConstraint(TOperationId opId, const TTxTransaction& tx, TOperationContext& context) {
+        Y_ABORT_UNLESS(tx.GetOperationType() == NKikimrSchemeOp::EOperationType::ESchemeOpCreateSetConstraint);
+        Y_UNUSED(context);
+        auto tablePath = NKikimr::JoinPath({tx.GetWorkingDir(), tx.GetSetColumnConstraintsRequest().GetTableName()});
+        TString error = "CreateSetConstraint is not implemented. TablePath = '" + tablePath + "'";
+        return {CreateReject(opId, NKikimrScheme::EStatus::StatusPreconditionFailed, std::move(error))};
+    }
```

Comment:
Тут и в других ydb/core/tx/schemeshard/schemeshard__operation_create_set_constraint*: когда появится тело этой и других операций, оно будет достаточно объемным и терять еще один сдвиг вправо может быть слишком накладно. 
```suggestion

TVector<ISubOperation::TPtr> CreateSetConstraint(TOperationId opId, const TTxTransaction& tx, TOperationContext& context) {
    Y_ABORT_UNLESS(tx.GetOperationType() == NKikimrSchemeOp::EOperationType::ESchemeOpCreateSetConstraint);
    Y_UNUSED(context);
    auto tablePath = NKikimr::JoinPath({tx.GetWorkingDir(), tx.GetSetColumnConstraintsRequest().GetTableName()});
    TString error = "CreateSetConstraint is not implemented. TablePath = '" + tablePath + "'";
    return {CreateReject(opId, NKikimrScheme::EStatus::StatusPreconditionFailed, std::move(error))};
}

```